$NetBSD: patch-ac,v 1.7 2006/08/30 15:58:25 rillig Exp $

Solaris has <getopt.h>, but the plain getopt() function is defined in
<unistd.h>, so it needs both headers.

--- configure.orig	2011-07-21 14:52:29.000000000 +0000
+++ configure
@@ -28383,11 +28383,10 @@ extern "C" {
 				   #endif
                                    #if HAVE_GETOPT_H
                                    #include <getopt.h>
-                                   #else
+                                   #endif
                                    #if HAVE_UNISTD_H
                                    #include <unistd.h>
                                    #endif
-                                   #endif
                                    }
 int
 main ()
